### PR TITLE
add workflow figs

### PR DESCRIPTION
**Pull request by monopole**

### CODE REVIEW

File: cmd/kinflate/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # kinflate
 
-[_kubectl apply_]: glossary.md#apply
-[DAM]: glossary.md#declarative-application-management
-[workflows]: workflows.md
+[_kubectl apply_]: docs/glossary.md#apply
+[DAM]: docs/glossary.md#declarative-application-management
+[workflows]: docs/workflows.md
```
Comment (droot) [inline cmd/kinflate/README.md]: +1 on moving these under docs dir.

File: cmd/kinflate/docs/workflows.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+## No local files
+
+One just installs some [OTS] configuration from the web,
+[applying] it directly to the cluster.
+
+This assumes a configuration specified at https://github.com/kinflate/ldap.
+
+> ```
+> kinflate inflate -f https://github.com/kinflate/ldap |\
```
Comment (droot) [inline cmd/kinflate/docs/workflows.md]: We don't support manifest off the HTTP URL yet, so we should take this out ?

File: cmd/kinflate/docs/workflows.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+> ```
+> kinflate inflate -f ~/myldap | kubectl apply -f -
+> ```
+
+The myldap dir contains a `Kube-manifest.yaml`
+referencing the base via URL, and a `deployment.yaml`
+resource [patch] that increases the replica count
+specified in the base.
+
+## Bespoke configuration
```
Comment (droot) [inline cmd/kinflate/docs/workflows.md]: we should probably move this to the top ?

File: cmd/kinflate/docs/workflows.md (hunk 2 above)
Comment (monopole) [reply cmd/kinflate/docs/workflows.md]: It's so easy to add though 😃

**Comment by mengqiy** [comment]
The figures looks great!
BTW don't forget to squash the commits before merging. They have the same name.

**Pull request merged**